### DISCUSSION TITLE
part-ii/2013-75/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#368] [Question comments]
Part II 2013 Graph Theory: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2013-75/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part follows from writing out the matrix $A - tI$ and expanding out to find the coefficients of the terms with the highest powers of $t$, e.g. the products that contribute to the $t^{n-2}$ term are exactly those with all but two of the diagonal entries chosen, and we get a contribution of $-1$ if and only if the two rows not chosen correspond to adjacent vertices; we need to take into account the fact that we have made one transposition here).

The next part was lectured, but I'll write it out anyway:

$$
\lambda_1 = \max_{\mathbf{x} \in \mathbb{R}^n, |\mathbf{x}| = 1} <\mathbf{x},A\mathbf{x}> \geq \frac{1}{n} <\mathbb{1},A\mathbb{1}> = \frac{1}{n}<\mathbb{1},\begin{pmatrix} d(1) \\ \vdots \\ d(n) \end{pmatrix}> = d
$$

We have that $\lambda_1 + \dots + \lambda_n = 0$, since this is equal to the trace of $A$ (which we can see is equal to $0$ by looking at the characteristic polynomial, since the trace is equal to $-c_1$, or indeed by looking at the adjacency matrix directly because we know we have zeros on the diagonal since we have no loops). 

Next, $c_2 = \prod_{i<j} \lambda_i \lambda_j$, so $\lambda_1^2 + \dots + \lambda_n^2 = (c_1)^2 - 2c_2 = 0 - 2 (-\frac{nd}{2}) = nd$ as required. 

To infer that $\lambda_1 \leq \sqrt{d(n-1)}$, it is sufficient to prove that $\lambda_1^2 \leq nd - d$, so by the previous result it is sufficient to prove that $\lambda_2^2 + \dots + \lambda_n^2 \geq d$. But we know that $\sum_{i=1}^{n} \lambda_i = 0$ and $\lambda_1 \geq d$, so $\sum_{i=2}^{n} \lambda_i \leq -d$. I think the result should follow quickly from here, though I can't immediately see how to do it (I'm not sure it's completely trivial).

Yes, such a graph exists for each $n$; we can take $G = K_{n-1}$ (we have $\lambda_1 = d = n - 1$, and $\lambda_2 = \dots = \lambda_n = -1$).

</details>
